src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/behandling/BehandlingsstatistikkService.kt

```diff
@@ -0,0 +1,207 @@
+                ?: henvendelseTidspunkt.atZone(europeOslo),
+            endretTid = hendelseTidspunkt.atZone(europeOslo),
+            tekniskTid = zonedNow(),
+            behandlingStatus = hendelse.name,
+            opprettetAv = maskerVerdiHvisStrengtFortrolig(
+                strengtFortroligAdresse,
+                saksbehandlerId,
+            ),
+            saksnummer = saksbehandling.eksternFagsakId.toString(),
+            mottattTid = henvendelseTidspunkt.atZone(europeOslo),
+            saksbehandler = maskerVerdiHvisStrengtFortrolig(
+                strengtFortroligAdresse,
+                saksbehandlerId,
+            ),
+            ansvarligEnhet = maskerVerdiHvisStrengtFortrolig(
+                strengtFortroligAdresse,
+                sisteOppgaveForBehandling?.tildeltEnhetsnr ?: MASKINELL_JOURNALFOERENDE_ENHET,
+            ),
+            behandlingMetode = behandlingMetode?.name ?: "MANUELL",
+            behandlingÅrsak = saksbehandling.årsak.name,
+            avsender = "NAV Tilleggstønader",
+            behandlingType = BehandlingType.valueOf(saksbehandling.type.name).toString(),
+            sakYtelse = saksbehandling.stønadstype.name,
+            behandlingResultat = saksbehandling.resultat.name,
+            resultatBegrunnelse = null, // TODO er fritekstfelt, og er ikke ønsket i statistikk før enum er implementert
+            ansvarligBeslutter =
+            if (!beslutterId.isNullOrEmpty()) {
+                maskerVerdiHvisStrengtFortrolig(
+                    strengtFortroligAdresse,
+                    beslutterId.toString(),
+                )
+            } else {
+                null
+            },
+            vedtakTid = if (Hendelse.VEDTATT == hendelse) {
+                hendelseTidspunkt.atZone(europeOslo)
+            } else {
+                null
+            },
+            ferdigBehandletTid = if (Hendelse.FERDIG == hendelse) {
+                hendelseTidspunkt.atZone(europeOslo)
+            } else {
+                null
+            },
+            totrinnsbehandling = totrinnskontrollErGodkjent(behandlingId),
+            sakUtland = mapTilStreng(saksbehandling.kategori),
+            relatertBehandlingId = relatertEksternBehandlingId,
+            versjon = Applikasjonsversjon.versjon,
+            vilkårsprøving = emptyList(), // TODO: Implementer dette i samarbeid med Team SAK. Ikke kritisk å ha med i starten.
+            revurderingÅrsak = null, // TODO aktiver når revurdering er implementert
+            revurderingOpplysningskilde = null, // TODO aktiver når revurdering er implementert
+        )
+    }
+
+    private fun settBeslutterId(
+        hendelse: Hendelse,
+        behandlingId: UUID,
+    ): String? {
+        val beslutterId = if (hendelse.erBesluttetEllerFerdig()) {
+            totrinnskontrollService.hentBeslutter(behandlingId)
+        } else {
+            null
+        }
+        return beslutterId
```

Comment:
Veldig pirk, men synes dette er mer lettlest 🤓 
```suggestion
        return if (hendelse.erBesluttetEllerFerdig()) {
            totrinnskontrollService.hentBeslutter(behandlingId)
        } else {
            null
        }
       
```